utils/service_discovery/sd_docker_backend.py

```diff
@@ -487,13 +488,14 @@ def _get_config_templates(self, identifier, **platform_kwargs):
 
         return templates
 
-    def _fill_tpl(self, state, c_id, instance_tpl, variables, tags=None):
+    def _fill_tpl(self, state, c_id, instance_tpl, variables, c_tags=None):
         """Add container tags to instance templates and build a
            dict from template variable names and their values."""
         var_values = {}
         c_image = state.inspect_container(c_id).get('Config', {}).get('Image', '')
 
-        # add default tags to the instance
+        # add only default c_tags to the instance to avoid duplicate tags from conf
+        tags = copy.copy(c_tags)
```

Comment:
Good catch! Thanks for digging in this issue.

If that's the only place we use it I'd rather not import copy and do `tags = c_tags[:]` instead. It will create a new list with all the elements of c_tags.